app/scoring/tests/test_scores_routes.py

```diff
@@ -0,0 +1,34 @@
+import uuid
+import os.path
+import pytest
+from flask import url_for
+from app.common.static import get_dict_from_json_file
+
+JSON_DIRECTORY = (
+    "cypress/fixtures"  # TODO: move to suitable python directory and make robust
```

Comment:
Regarding this TODO, I haven't moved the files yet, as they are used by other cypress tests yet to be rewritten.

Reply:
Good idea, I've copied them to under common/, as they are also used by other e2e tests.